src/sources/kubernetes_logs/path_helpers.rs

```diff
@@ -34,7 +38,15 @@ pub(super) fn build_pod_logs_directory(
 ///
 /// Inspired by <https://github.com/kubernetes/kubernetes/blob/31305966789525fca49ec26c289e565467d1f1c4/pkg/kubelet/kuberuntime/helpers.go#L186>
 pub(super) fn parse_log_file_path(path: &str) -> Option<LogFileInfo<'_>> {
-    let mut components = path.rsplit('/');
+    let delimiter;
+
+    if path.chars().nth(4) == Some('/') {
```

Comment:
Rather than looking at the filepath to try to determine the host system, could we instead use `std::path::MAIN_SEPARATOR`?

Reply:
Yes, thanks for the idea, I changed it to MAIN_SEPARATOR. I think this will be right.